page89.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+age = 15
+
+if age < 2:
+    print("You're a baby.")
+
```

Comment:
No blank lines needed between the conditions. This comment applies to other lines below this.